packages/data-broker/src/services/dhis/DhisService.js

```diff
@@ -133,7 +138,7 @@ export class DhisService extends Service {
   }
 
   getPullAnalyticsForApiMethod = options => {
-    const { programCodes } = options;
+    const { programCodes, dhisDataType } = options;
```

Comment:
`dhisDataType: 'Indicator'` is specified in the config of the data_source. I chose this way instead of setting type column indicator because I feel that indicator on DHIS2 is not much different from a data element (it still returns analytics format) except that we cannot delete/push data to it. If we set type column = indicator, I feel that it is going to create unnecessary distinguishment between them in a some places.